packages/visual-dev/CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [1.2.0] - 2022-11-
+## [1.2.0] - 2022-11-24
```

Comment:
Just calling this out now, we need to make sure we remember to tag all these release